### DISCUSSION TITLE
CSS slowly loaded

### DISCUSSION THREAD

**Discussion by ghost** [solidjs/solid-start#1571] [Q&A] [answered]
i just trying solid start today and found whenever i reload the page, CSS is slowly loaded

https://github.com/solidjs/solid-start/assets/68492618/c5451459-c3af-4754-8df8-f5d733bd5d81

how to fix this?

**Comment by bclewer** [comment]
I am also seeing the same issue via solid start

**Reply by dhairya-cse** [reply, marked as answer]
It was happening with me as well but fixed when updated the dependencies
```json
  "dependencies": {
    "@solidjs/start": "^1.0.4",
    "solid-js": "^1.8.18",
    "vinxi": "^0.3.14"
  },
  ```

**Comment by ryansolid** [comment]
Yeah sorry there was a bug introduced Vinxi 0.3.12 that only impacted Dev that took a little bit of time to be fixed... We have a special system to load styles in Dev because that flash of unstyled content is the Vite default when doing SSR and a different bug fix had broken it. It should be all good now.
